recipes/mender-shell/debian-master/rules

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC=${CC} \
```

Comment:
Nit:
Inconsistent quoting. Some are, some arent, of the variables expanded.

Reply:
Thanks. I'll fix here and in the mender-client recipe